packages/storybook/stories/UseCases.stories.mdx

```diff
@@ -114,10 +114,10 @@ These client libraries also require the context for the call or chat they will j
 
 ## Platform support
 
-| SDK    | Windows            | macOS                | Ubuntu   | Linux    | Android  | iOS        |
-| ------ | ------------------ | -------------------- | -------- | -------- | -------- | ---------- |
-| UI SDK - Calling | Chrome\*, new Edge | Chrome\*, Safari\*\* | Chrome\* | Chrome\* | Chrome\*\* | Safari\*\*\* |
-| UI SDK - Chat    | Firefox\*, Chrome\*, new Edge | Firefox\*, Chrome\*, Safari\* | Chrome\* | Chrome\* | Safari\*|
+| SDK              | Windows                       | macOS                         | Ubuntu   | Linux    | Android    | iOS          |
```

Comment:
Does this look ok visually?

Reply:
yup this looks okay to me, not sure how it's changed